README.md

```diff
@@ -186,8 +203,29 @@ export default {
 ### Provide human-readable labels for URIs on the subject page
 Check the README of the [resource label service](https://github.com/lblod/resource-label-service/#db-information) how to insert human-readable labels in the triplestore. Once the labels are available, they will be automatically picked up by ember-metis.
 
-### Customize the resource view
-To allow you to customize the appearance of the resource pages, the addon offers two components:
+### Customize the appearance
+
+#### Customize a few components
+To allow you to customize the appearance of the resource page, the template is composed of multiple small components. You can override any of them to change the component's appearance on all resource pages.
+
+For instance, if you want to show different levels of headings with different colors, add the following to your `app/components/metis/heading.hbs`:
```

Comment:
Overriding components this way is a bad practice (even if it was once popular..). We shouldn't be recommending this and it will stop working if we switch to .gjs components in the future.

A better way would be to let consuming apps register components they want to override somehow and clearly document which arguments that component will receive.